### PR TITLE
Improvements and changes (see description)

### PR DESCRIPTION
**Pull request by benj-o**
- Take MOTD's straight off 2b2t and display them like 2b2t does!
- Added 2b2t's server icon image.
- Changed various messages to make them less wordy.

I get the messages are of personal preference but I notice that they all tend to be 90% dead space. I've just compacted them and removed unnecessary parts which you can try out in 20 secs on discord with this updated main script.

### CODE REVIEW

**Comment by rob9315** [comment]
please add `xmlhttprequest` to the dependencies so this actually works. 
I like the other changes.

**Comment by benj-o** [comment]
I'm sorry about that, would adding "xmlhttprequest": "1.8.0" to dependencies list fix this?

**Comment by rob9315** [comment]
Yes, that should fix it

**Comment by benj-o** [comment]
Perfect! Added (I believe)

File: main.js [hunk 1]
```diff
@@ -248,15 +253,22 @@ function join() {
 						let totalWaitTime = getWaitTime(queueStartPlace, 0);
 						let timepassed = getWaitTime(queueStartPlace, positioninqueue);
 						let ETAmin = (totalWaitTime - timepassed) / 60;
-						server.motd = `Place in queue: ${webserver.queuePlace} ETA: ${webserver.ETA}`; // set the MOTD because why not
+
+                                                var req = new XMLHttpRequest();
```
Comment (u9g) [inline main.js:257]: why not use minecraft-protocol for this request?

Comment (benj-o) [reply main.js:257]: No reason other than unfamiliarity with said package

Comment (cristiioan) [reply main.js:257]: Definelty minecrraft-protcol will be cool.

**Comment by themoonisacheese** [comment]
closing as abandonned.

**Pull request closed without merging**